src/Microsoft.XmlSerializer.Generator/src/System/Xml/Serialization/SoapAttributesExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
```

Comment:
Add copy right header at the beginning. It'd be good to use CodeFormatter to scan the project and fix the issues like this.